src/lib/state/DY.Lib.State.PrivateKeys.fst

```diff
@@ -16,17 +17,9 @@ open DY.Lib.State.Map
 
 (*** Private keys types & invariants ***)
 
-[@@ with_bytes bytes]
-type private_key_type =
-  | PkDec: [@@@with_parser #bytes ps_string] usage:string -> private_key_type
-  | Sign: [@@@with_parser #bytes ps_string] usage:string -> private_key_type
-
-%splice [ps_private_key_type] (gen_parser (`private_key_type))
-%splice [ps_private_key_type_is_well_formed] (gen_is_well_formed_lemma (`private_key_type))
-
 [@@ with_bytes bytes]
 type private_key_key = {
-  ty:private_key_type;
+  ty:public_key_type;
```

Comment:
Indeed. What about `long_term_key_type`? It would go well with the fact they start with `LongTerm`.
We could also do `asymetric_key_type` I like it less.

Reply:
I think that does work, since I don't expect there to be a cause to have a general long-term symmetric key type. Maybe someone else will have an idea as well?

Reply:
Switched to `long_term_key_type` (but can switch to something else if needed)